tests/tabnote_tests.ts

```diff
@@ -384,7 +383,7 @@ function drawStemsUpThrough(options: TestOptions, contextBuilder: ContextBuilder
     return tabNote;
   });
 
-  ctx.setFont('sans-serif', 10, 'bold');
+  ctx.setFont(Font.SANS_SERIF, 10, 'bold');
```

Comment:
Unfortunately in my Font PR branch, I have reverted to using things like 'sans-serif' in many of these test cases. This allowed me to achieve zero visual diffs from master. After the merge, I will submit a PR for using the Font constants, which will introduce a lot of very subtle visual diffs.

Reply:
This one is causing no visual diff and you left `Font.SANS_SERIF`. Is it OK to leave it or you want it reverted?

Reply:
Approved, no change required.